siliconcompiler/schema/schema_cfg.py

```diff
@@ -913,24 +938,32 @@ def schema_datasheet(cfg, design='default', name='default', mode='default'):
     # AC Timing
     metrics = {'tsetup': ['setup time', (1e-9, 2e-9, 4e-9), 's'],
                'thold': ['hold time', (1e-9, 2e-9, 4e-9), 's'],
+               'tdelay': ['propagation delay', (1e-9, 2e-9, 4e-9), 's'],
                'trise': ['rise transition', (1e-9, 2e-9, 4e-9), 's'],
-               'tfall': ['fall transition', (1e-9, 2e-9, 4e-9), 's'],
-               'tperiod': ['minimum period', (1e-9, 2e-9, 4e-9), 's'],
-               'tpulse': ['pulse width', (1e-9, 2e-9, 4e-9), 's'],
-               'tjitter': ['rms jitter', (1e-9, 2e-9, 4e-9), 's'],
-               'dutycycle': ['duty cycle', (45, 50, 55), '%']}
+               'tfall': ['fall transition', (1e-9, 2e-9, 4e-9), 's']}
 
-    for item, val in metrics.items():
-        scparam(cfg, ['datasheet', design, 'pin', name, item, mode],
-                unit=val[2],
+    for i, v in metrics.items():
```

Comment:
why not do: `desc, limits, unit = v` that would be a little clearer in the code below instead of the array indexing.